File: javascript/7kyu/Jumping Number Special Numbers Series 4.js
// https://www.codewars.com/kata/5a54e796b3bfa8932c0000ed

// Definition

// Jumping number is the number that All adjacent digits in it differ by 1.
// Task

// Given a number, Find if it is Jumping or not . 

function jumpingNumber(n){
    let arr = n.toString().split('')
    for(i=0; i < arr.length-1; i++){
      if(Math.abs(arr[i] - arr[i+1]) !== 1 ){
        return 'Not!!'
      }
    }
    return 'Jumping!!'
}

File: java/7kyu/Jumping Number Special Numbers Series 4.java
// https://www.codewars.com/kata/5a54e796b3bfa8932c0000ed

// Definition

// Jumping number is the number that All adjacent digits in it differ by 1.
// Task

// Given a number, Find if it is Jumping or not . 

public class Solution
{
    public static String jumpingNumber(int number) {
        while (number > 10) {
            int previous = number % 10;
            number = number / 10;
            if (previous - number % 10 != 1 && previous - number % 10 != -1) {
                return "Not!!";
            }
        }
        return "Jumping!!";
    }
  }